completions/quick-lint-js.bash

```diff
@@ -14,14 +18,18 @@ _quick-lint-js () {
                         return
                         ;;
                 --output-format=*)
-                        COMPREPLY=($(compgen -W 'gnu-like vim-qflist-json emacs-lisp' -- "${cur#*=}"))
+                        COMPREPLY=($(compgen -W '${_quick_lint_js_output_formats}' -- "${cur#*=}"))
```

Comment:
Doesn't `'` (single quote) disable variable expansion? Don't we want `"${_quick_lint_js_output_formats}"` (double quotes) instead?

Reply:
<img width="855" alt="Screenshot 2023-04-18 at 11 21 49 am" src="https://user-images.githubusercontent.com/38635403/232649339-3a596394-ddff-4c0c-9cc0-ed2e0e432aa3.png">

single quote seems like working on bash 5.2, but I think double quote would cause less confusion because I believe single quote wasn't working in the past.
I will change them to double quote 👍